types/webpack/index.d.ts

```diff
@@ -18,6 +18,7 @@
 //                 Grgur Grisogono <https://github.com/grgur>
```

Comment:
I know this PR isn't updating the types for a newer version of webpack, but I think it would still be a good idea to bump up the version at the top of this file to 4.41. This PR has a few breaking changes and if someone does get broken they can go back to the 4.39 type definitions.

Reply:
Thanks for the feedback 👍
Agree with that and increased the version to `4.41` accordingly.

I've also checked if there were new changes to introduced in this release to the affected types but nothing changed there.